tools/devcluster.yaml

```diff
@@ -2,9 +2,9 @@
 startup_input: "p"
 
 commands:
-  p: make -C harness clean build  # rebuild Python
-  w: make -C webui build          # rebuild Webui
-  c: make -C docs build           # rebuild doCs
+  p: make -C bindings build && make -C harness clean build  # rebuild Python
+  w: make -C bindings build && make -C webui build          # rebuild Webui
+  c: make -C docs build  # rebuild docs
```

Comment:
the capital letter was the mnemonic, same with Python and Webui

Reply:
ha

Reply:
it works better in Warcraft III where they use yellow letters in a white font.  But with ascii, you have to work with what you got